metapackage_dependencies.rosinstall

```diff
@@ -7,4 +7,4 @@
 - git:
     uri: https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge.git
     local-name: uwrt_mars_rover/uwrt_mars_rover_hw_bridge/lib/uwrt_mars_rover_hw_bridge/
-    version: 99871826200512a7066e2973bf2460a95e4f7b3b
+    version: fix-ros-build
```

Comment:
Will change this to commit once https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge/pull/12 merges